README.md

```diff
@@ -41,6 +42,7 @@ The different options are:
 * `titleExtra`: Any additional text relating to the title, e.g. site _section_.
 * `summary`: Summary text to be shared.
 * `relatedTwitterAccounts`: Comma-separated list of Twitter accounts to encourage the user to follow. See [Twitter intents](https://dev.twitter.com/docs/intents) for more info.
+* `locationOfShareComponent`: A unique identifier for the share component which is used to track shares when multiple components exist on the page.
```

Comment:
@GlynnPhillips sorry that was not at all clear of me 🙈 I just meant in the docs 
```suggestion
* `locationOfShareComponent` (optional): A unique identifier for the share component which is used to track shares when multiple components exist on the page.
```

Reply:
I see that was partly what I thought you might have meant but wanted to double check. I am happy to add this but many of those attributes are also optional so by adding it for one makes it a bit more confusing. Shall I do it for all optional attributes whilst I am at it?

Reply:
I went wild and released, but that would be nice